test/test_flake8.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
```

Comment:
If we're copying this from somewhere, do we keep the C year? If not, could we just make it BSD (given how trivial this is)?

Reply:
Fair point. I grabbed these from `rqt_reconfigure`, so I'll revert to the original year, 2019.

Reply:
98fa4fc